rubin_scheduler/scheduler/example/example_scheduler.py

```diff
@@ -1363,15 +1398,13 @@ def main(args):
 
     repeat_night_weight = None
 
-    # Use the Almanac to find the position of the sun at the start of survey
-    almanac = Almanac(mjd_start=mjd_start)
-    sun_moon_info = almanac.get_sun_moon_positions(mjd_start)
-    sun_ra_start = sun_moon_info["sun_RA"].copy()
+    observatory = ModelObservatory(nside=nside, mjd_start=mjd_start)
+    conditions = observatory.return_conditions()
 
     footprints = make_rolling_footprints(
         fp_hp=footprints_hp,
-        mjd_start=mjd_start,
-        sun_ra_start=sun_ra_start,
+        mjd_start=conditions.mjd_start,
+        sun_ra_start=conditions.sun_ra_start,
```

Comment:
Doesn't this undo the "use the Almanac instead of the full Conditions" modification, so that we can set up the scheduler without needing skybrightness files immediately?